README.md

```diff
@@ -89,6 +89,12 @@ bundle install
 # Fetch tests
 bundle exec thor kitchen:fetch-remote-tests
 
+# download a MySQL-installation role
+ansible-galaxy install -p roles/ bennojoy.mysql
+
+# change password in MySQL-installation role to match the one from testing
+sed -i 's/foobar/iloverandompasswordsbutthiswilldo/g' roles/bennojoy.mysql/defaults/main.yml
```

Comment:
That's the plan.
I don't want to include the role itself, so the `ansible-galaxy`-command installs the role, then you can test it.
For normal usage of the hardening-role this role is not required.